src/hid-sensor.cpp

```diff
@@ -20,6 +20,17 @@ static const std::map< rs2_stream, uint32_t > stream_and_fourcc
         { RS2_STREAM_ACCEL, rs_fourcc( 'A', 'C', 'C', 'L' ) },
         { RS2_STREAM_GPIO,  rs_fourcc( 'G', 'P', 'I', 'O' ) } };
 
+/*For gyro sensitivity - FW gets 0 for 61 milligegree/s/LSB resolution
+ 0.1 for 30.5 milligegree/s/LSB 
+ 0.2 for 15.3 milligegree/s/LSB 
+ 0.3 for 7.6 milligegree/s/LSB 
+ 0.4 for milligegree/s/LSB */
+static const std::map< float, double > gyro_sensitivity_convert
```

Comment:
Will this stay the same for D500? Maybe add a TODO comment if there is work to be done later

Reply:
Added the comment

Reply:
Do we still need this? FW supposed to support the enum values